src/main/java/uk/ac/ebi/embl/api/validation/ValidationScope.java

```diff
@@ -77,8 +74,11 @@ public int getAssemblyLevel() {
   }
 
   public enum Group {
+    PIPELINE,
```

Comment:
Suggest to add a few comments ->

/** Webin submissions processed by pipelines. */
PIPELINE

/** Non-Webin submissions processed by putff. */
PUTFF